presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/query/TestPinotQueryGeneratorSql.java

```diff
@@ -182,6 +182,14 @@ public void testSelectionWithOrderBy()
                 defaultSessionHolder,
                 ImmutableMap.of());
 
+        TopNNode topNNode = topN(planBuilder, 50L, ImmutableList.of("fare", "city"), ImmutableList.of(false, false), tableScanNode);
+        testPinotQuery(
+                pinotConfig,
+                project(planBuilder, topNNode, ImmutableList.of("regionid", "city")),
+                "SELECT regionId, city FROM realtimeOnly ORDER BY fare DESC, city DESC LIMIT 50",
```

Comment:
How about an ascending order test query here for the Pinot sql case ... Since pinot pql only supports order by desc I believe. (Not sure at all !)

Reply:
I think it supports as I see asc cases above, made sort order for one of the columns ascending